src/helpers/identities.ts

```diff
@@ -162,7 +162,7 @@ const getAvailableAddresses = (
 };
 
 /**
- * Returns the list of all the identities for the account. For each identity a type
+ * Returns the list of all the identities for the account. For each identity a typnpm e
```

Comment:
what does typnpm mean?

Reply:
your guess is as good as mine :) 
thanks for spotting it, fixing it now